README.md

```diff
@@ -28,15 +30,15 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+It doesn't make a difference, they both basically do the same thing.
 ```
 
```

Comment:
👍  any HTML attribute prefixed with data- will be ignored by the html validator, and can therefore be used to make sure an Angular app's html can validate